src/submit_message.js

```diff
@@ -71,45 +73,50 @@ export default function (data) {
     var jsonBody = JSON.parse(r.body);
     var messageId = jsonBody.id;
 
-    // Polling message status
-    var maxRetries = 20;
-    var retryCount = 0;
-    var messageSubmitted = false;
-    do {
-        sleep(1)
-        console.log('Polling Message Status: ' + r.status + ' retry: ' + (retryCount + 1));
-
-        // Get Message Status
-        tag = {
-            pagoPaMethod: "GetMessageStatus",
-        };
-        url = `${urlBasePath}/api/v1/messages/${fiscalCode}/${messageId}`;
-        r = http.get(url, headersParams, {
-            tags: tag,
-        });
-        check(r, { 'Get Message Status is 200': (r) => (r.status === 200) },
-            tag
-        );
-
-        // If status is "PROCESSED" then the message was correctly submitted
-        var jsonBody = JSON.parse(r.body);
-        if (r.body && jsonBody.status === 'PROCESSED') {
-            messageSubmitted = true;
-            break;
+    // Sample Message Status
+    var r = Math.floor(Math.random() * 100) + 1;
+    if (r <= samplingRate) {
+        // Polling message status
+        var maxRetries = 20;
+        var retryCount = 0;
+        var messageSubmitted = false;
+        do {
+            sleep(1)
```

Comment:
```suggestion
            sleep(retryCount++)
```
add incremental retry